### PR TITLE
docs(halyard): Adds small architecture diagram to the Halyard codelab

### PR DESCRIPTION
**Pull request by ttomsu**
@spinnaker/google-reviewers FYI

### CODE REVIEW

**Review by ewiseblatt — COMMENTED** [review]
I know this is just an example, however would it be "normal" to be deploying into the same cluster as Spinnaker or would you typically keep the cluster under management separate? Is the codelab clear about that so when people copy it as a starting point they are headed down a good path?

**Review by lwander — APPROVED** [review]

**Comment by lwander** [comment]
I think this is fine because it's scoped to the codelab, maybe a disclaimer that this is only one possible outcome could help.

File: guides/tutorials/codelabs/halyard-getting-started/index.md [hunk 1]
```diff
@@ -20,6 +20,11 @@ In our scenario, we want to create a Spinnaker instance and set it up as follows
 
 For this exercise we will be operating entirely within one GCP project, and use Google Container Engine (GKE) as our Kubernetes cluster. 
 
+{% include figure
```
Comment (skim1420) [inline guides/tutorials/codelabs/halyard-getting-started/index.md:23]: Just curious, why not just embed the image via markdown, i.e. `?`

Comment (skim1420) [reply guides/tutorials/codelabs/halyard-getting-started/index.md:23]: Also, does the markup-based diagramming not support stuff like this?

Comment (ttomsu) [reply guides/tutorials/codelabs/halyard-getting-started/index.md:23]: I could - I wanted a caption though, and the `include figure` provides that with less fuss. Take a look at https://github.com/spinnaker/spinnaker.github.io/blob/master/_includes/figure

**Pull request merged**